packages/block-library/src/preformatted/edit.native.js

```diff
@@ -28,10 +28,12 @@ export function PreformattedEdit( props ) {
 		...( style?.fontSize && { fontSize: style.fontSize } ),
 		...( style?.color && { color: style.color } ),
 	};
+	const hasBaseColors =
+		style?.baseColors && Object.entries( style?.baseColors ).length !== 0;
```

Comment:
Nitpick: Since `style.baseColors` existence is already checked in the first condition, we don't really need to use optional chaining in the second one.
```suggestion
		style?.baseColors && Object.entries( style.baseColors ).length !== 0;
```

Reply:
Thanks! I'll update it 👍